src/tests/compiler/ast/doc/base/jsDocPropertyLikeTagTests.ts

```diff
@@ -9,18 +9,29 @@ describe(nameof(JSDocPropertyLikeTag), () => {
         return { descendant: info.sourceFile.getFirstDescendantOrThrow(TypeGuards.isJSDocPropertyLikeTag), ...info };
     }
 
+    describe(nameof<JSDocPropertyLikeTag>(d => d.getTypeExpressionNode), () => {
+        function doTest(text: string, expected: string) {
+            const { descendant } = getInfo(text);
+            expect(descendant.getTypeExpressionNode()!.getTypeNode().getText()).to.equal(expected);
```

Comment:
Could you add a test for when this doesn't exist? I usually use this same function, change `expected: string | undefined`, then change the body appropriately, but anything works.